marklogic-data-hub/src/main/resources/hub-internal-artifacts/flows/default-ingestion.flow.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name" : "default-ingestion",
-  "description" : "This is the default flow for ingesting into data hub",
+  "name": "default-ingestion",
+  "description": "This is the default flow for ingesting into data hub",
   "options": {
   },
-  "steps" : {
-	  "1": {	 
-	   "stepDefinitionType": "ingestion",
-	   "stepDefinitionName": "default-ingestion",
-	   "retryLimit": 0,
-	   "options": {
-        "outputFormat" : "json",
-        "collections" : "default-ingestion"
-     }
-	  }
-	}
+  "steps": {
+    "1": {
+      "stepDefinitionType": "ingestion",
+      "stepDefinitionName": "default-ingestion",
+      "retryLimit": 0,
+      "options": {
+        "outputFormat": "json",
+        "collections": "default-ingestion"
```

Comment:
Can we have the collections as an array to make it consistent across all the flow and step def artefacts